bin/convert-argv.js

```diff
@@ -18,7 +18,7 @@ module.exports = function(yargs, argv, convertOptions) {
 	}
 	if(argv.p) {
 		argv["optimize-minimize"] = true;
-		argv["define"] = [].concat(argv["define"] || []).concat("process.env.NODE_ENV=\"production\"");
+		argv["define"] = Array.prototype.concat(argv["define"] || []).concat("process.env.NODE_ENV=\"production\"");
```

Comment:
It does, but the prototype is empty and concat is an immutable method so it's okay.